proto/process/agent.proto

```diff
@@ -294,6 +294,7 @@ message CollectorManifest {
 message CollectorManifestCRD {
 	CollectorManifest manifest = 1;
 	repeated string tags = 2;
+	map<string, string> collectedVersions = 3;
```

Comment:
Looking at the `CollectorManifest` message, it can contains many CRDs. Could you give an example of expected values for this map? How do we handle when a single CRD has many versions deployed to a cluster?

https://github.com/DataDog/agent-payload/blob/925b62fb501d277c38e32e5f3a34db4a9db977c3/proto/process/agent.proto#L275-L283